### PR TITLE
Refactor Building and Testing docs

### PR DESCRIPTION
**Pull request by dfarrell07**
Signed-off-by: Daniel Farrell <dfarrell@redhat.com>

### CODE REVIEW

**Comment by dfarrell07** [comment]
Some of these docs are currently "aspirational", as in I want to make reality as described by the docs but it's not quite there yet. For example, common `build`/`images` targets in each repo.

**Review by nyechiel — APPROVED** [review]
LGTM

File: src/content/contributing/building_testing/_index.en.md [hunk 1]
```diff
@@ -4,163 +4,99 @@ date: 2020-03-18T16:03:26+01:00
 weight: 10
 ---
 
-Submariner strives to be an open, welcoming community for developers.
-Substantial tooling is provided to ease the contribution experience.
+Submariner strives to be an open, welcoming community. Substantial tooling is provided to ease the contribution experience.
 
 ## Standard Development Environment
 
 Submariner provides a standard, shared development environment suitable for all local work. The same environment is used in CI.
```
[the remark this answers is not in the document]
Comment (dfarrell07) [reply src/content/contributing/building_testing/_index.en.md:11]: I think this alternative has a "buried lead", as in we have to read to the end of the sentence to get to the key point. It's not important to the reader that the project is called "Shipyard", it's important that they can use this shared env locally.

[the remark this answers is not in the document]
Comment (dfarrell07) [reply src/content/contributing/building_testing/_index.en.md]: Yep, that's much nicer imho, thanks @tpantelis!

**Review by tpantelis — APPROVED** [review]

**Pull request merged**